src/topics/outputs.md

```diff
@@ -35,11 +35,16 @@ any subsequent elements are mandatory command line arguments
 :name: tar-job.yml
 ```
 
-Next, create a tar file for the example and invoke `cwl-runner` with the tool
-wrapper and the input object on the command line:
+Next, create a tar file for the example.
 
 ```{code-block} console
 $ touch hello.txt && tar -cvf hello.tar hello.txt
+hello.txt
+```
+
+And now invoke `cwl-runner` with the tool wrapper and the input object on the command line:
```

Comment:
An addition to the style guide, use "tool description" not "tool wrapper"

Reply:
TIL, will fix it and update the style guide too. Thanks!

Reply:
Done!

Reply:
descript**ion** not descript**or** :-)

Reply:
Yeah, I went to do a quick self-edit of what I wrote, and realized it. Then I tried fixing that after rebasing, and wrote “Use tool description, not tool description”. Then fixed that and messed up somewhere else. Argh. Hopefully fixed now :crossed_fingers: